src/main/java/org/webeid/security/util/SubjectCertificatePolicies.java

```diff
@@ -4,7 +4,10 @@
 
 public final class SubjectCertificatePolicies {
 
-    public static final ASN1ObjectIdentifier EST_MOBILE_ID_POLICY = new ASN1ObjectIdentifier("1.3.6.1.4.1.10015.1.3");
+    public static final ASN1ObjectIdentifier ESTEID_SK_2015_OLD_MOBILE_ID_POLICY =
+        new ASN1ObjectIdentifier("1.3.6.1.4.1.10015.1.3.3");
```

Comment:
Old policy had version suffix in the OID. You should probably check 1/2/3 or verify only first part of OID

Reply:
Alright, I'll do it with `certOid.startsWith(configuredOid)`